README.NewResourceFromTerraform.md

```diff
@@ -438,7 +438,14 @@ section under the Appendix.
        # Export the environment variables needed in the dynamic tests if you haven't done it.
        TEST_FOLDER_ID=123456789 go test -v -tags=integration ./pkg/controller/dynamic/ -test.run TestCreateNoChangeUpdateDelete -run-tests cloudschedulerjob -timeout 900s
     ```
-
+    
+1.  Our test framework can capture raw HTTP logs, helpful for debugging Kubernetes controller and GCP API interactions. Enable this by setting the ARTIFACTS environment variable:
+   
+    ```bash
+       # Save HTTP logs to /path/to/log.
+       ARTIFACTS=/path/to/log go test -v -tags=integration ./pkg/controller/dynamic/ -test.run TestCreateNoChangeUpdateDelete -run-tests basicpubsubsubscription -timeout 900s
```

Comment:
Could you clarify the "variables" that users should change? E.g.

\```
\# Change '/path/to/log' to the path where you want to save HTTP logs.
\# Change 'basicpubsubsubscription' to the folder name of the testdata.
\# Change '900s' to the timeout you want if needed.
\```

I recall getting the feedback a few times that contributors didn't know how to use the test command properly.

Reply:
Sounds good, updated!